packages/auth/src/grant/service.test.ts

```diff
@@ -149,7 +184,7 @@ describe('Grant Service', (): void => {
       const fetchedGrant = await grantService.getByContinue(
```

Comment:
```suggestion
      assert.ok(interactRef)
      const fetchedGrant = await grantService.getByContinue(
```
and remove the `as string` cast below